src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+    private static final int MAX_VALUE_FOR_FIGURES = 26;
+    private static final int MAX_VALUE_FOR_CIRCLE_RADIUS = 10;
+    private static final Random random = new Random();
```

Comment:
Random does not have to be a constant.
```suggestion
    private Random random = new Random();
```